bench-vortex/build.rs

```diff
@@ -0,0 +1,13 @@
+#![allow(clippy::unwrap_used)]
+#![allow(clippy::expect_used)]
+use std::env;
+use std::path::PathBuf;
+
+fn main() {
+    const DUCKDB_VERSION: &str = "v1.3.0";
+
+    let manifest_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
```

Comment:
Would it be easier to statically link in vortex-duckdb-ext and avoid consumers having to configure anything?

Reply:
As in, it pulls in symbols that are not part of the static lib itself.

Reply:
Plus linking large static libs is slow.